client/prisma/schema.prisma

```diff
@@ -0,0 +1,200 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Keywords {
+  id        String        @id @default(cuid()) // prisma generates unique id at runtime not implemented database level
+  name      String        @unique @db.VarChar(255)
+  createdAt DateTime      @default(now())
+  updatedAt DateTime      @updatedAt
+  deletedAt DateTime? //optional field
+  users     UserKeyword[]
+  jobs      JobKeyword[]
+  keywords  GigKeyword[]
+}
+
+model Users {
+  id              String            @id @default(cuid())
+  avatarUrl       String //suggestion: making this field optional
+  coverPhotoUrl   String //suggestion: making this field optional
```

Comment:
Oh yes please, I wanted these optional too. Maybe I missed out making it an optional field on the DB design. 

It's trivial but are you able a check constraint that it should never be an empty string? `""` while it is nullable.